flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -256,6 +254,9 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 	GPSPositionData gpsPosData;
 	BaroAltitudeData baroAltitude;
 	FlightStatusData flightStatus;
+
+	uint8_t *serial_buf = frsky_state->serial_buf;
```

Comment:
There's a bunch of additional lines to change, but I'll change them.  (It's not just the send calls, but all the buffer packing calls).

The optimizer can't do it because it's difficult to know if the frsky_state pointer has changed.  It is scoped to the module, and there is only one place that calls it... but the compiler can't know if when we call into another module if it re-enters that code there and changes frsky_state under us.